assets/sass/globals/forms.scss

```diff
@@ -171,6 +183,20 @@ $errorColor: #b00;
     }
 }
 
+.ff-field-pair {
```

Comment:
Adds the concept of a pair of form fields. Avoids needing to fight against grid class margins.